index.html

```diff
@@ -0,0 +1,847 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page Title</title>
+    <script src="https://kit.fontawesome.com/03bae28e4c.js" crossorigin="anonymous"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body class="body">
+
+
+    <section class="navsection">
+
+        <nav class="top-menu">
+            <ul>
+                <li class="resposive-logo">
+                    <img class="responsive-img-logo" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="triangle with all three sides equal" />
+                </li>
+                <li class="responsive"><a href="">News</a></li>
+                <li class="responsive"> <a href="">Events</a> </li>
+                <li class="responsive"> <a href="">Business</a></li>
+                <li class="responsive"> <a href="">AMAs</a></li>
+                <li class="responsive"><a href="">Spaces</a> </li>
+                <li class="responsive termscondition"><a href="">Terms & Conditions</a></li>
+                <li class="responsive"> <a href="">About</a></li>
+                <li class="responsive"> <a href="">Team</a> </li>
+                <li class="responsive"> <a href="">Advertise</a> </li>
+                <li class="responsive"> <a href="">Jobs</a> </li>
+                <li class="responsive"> <a href="">Contact</a> </li>
+                <li class="search-icon">
+                    <ul class="">
+                        <li class="forresponsive"> <a href=""><i class="fa fa-search"></i></a> </li>
+                        <li class="forresponsive"> <a href=""><i class="fa fa-bars"></i></a> </li>
+                    </ul>
+                </li>
+
+            </ul>
```

Comment:
#### Navigation bar
###### Large version

* [ ] It contains search icon at the rightmost side of sub-navigation row ([link to page](https://thenextweb.com/))

<img width="1440" alt="Screenshot 2020-01-03 at 16 27 06" src="https://user-images.githubusercontent.com/31304943/71731908-36c9ba80-2e46-11ea-9c3e-cc1071c4e300.png">

###### Medium version changes

* [ ] TWN logo appears at the leftmost part ([link to page](https://thenextweb.com/))

<img width="859" alt="Screenshot 2020-01-03 at 16 30 11" src="https://user-images.githubusercontent.com/31304943/71731985-6aa4e000-2e46-11ea-8e14-97c0c7da9e3a.png">